### PR TITLE
Fix number of files, total additions, and deletions on Diff pages

### PR DESCRIPTION
**Pull request by zeripath**
Use diff --shortstat to generate number of files, total additions and deletions on diff pages generated from git pages.

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

File: modules/git/repo_compare.go [hunk 1]
```diff
@@ -84,14 +87,112 @@ func (repo *Repository) GetCompareInfo(basePath, baseBranch, headBranch string)
...
+			func(ctx context.Context, cancel context.CancelFunc) error {
+				_ = stdoutWriter.Close()
+				buf := make([]byte, 1024)
+				var err error
+				for err == nil {
+					var n int
+					n, err = stdoutReader.Read(buf)
+					localBuf := buf[:n]
+					for {
+						if len(localBuf) <= 0 {
```
Comment (6543) [inline modules/git/repo_compare.go]:
```suggestion
						if len(localBuf) == 0 {
```

Comment (6543) [reply modules/git/repo_compare.go]: (fix lint)

**Review by lunny — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]
🏅 👏

**Comment by techknowlogick** [comment]
ping LG-TM

**Pull request merged**